frontend/view-edit-client/client-home/view-edit-demographics-info.component.tsx

```diff
@@ -95,7 +95,11 @@ export default function ViewEditDemographicsInfo(
               </tr>
               <tr>
                 <td>Annual income:</td>
-                <td>{currencyFormatter.format(client.householdIncome)}</td>
+                <td>
+                  {client.householdIncome === null
```

Comment:
Same here do a truthiness check.

Reply:
Actually on further thought, just remove your changes entirely. Household income is required. It will never be falsy